src/test/java/uk/gov/digital/ho/proving/income/IpsSearchTest.java

```diff
@@ -119,16 +128,19 @@ public void thatPassingIndividualReturnsSuccess() throws IOException {
                         "the expected value '03/07/2018'",
                 "03/07/2018",
                 ipsSearchPage.getIncomeToDate().getText());
-        assertTrue("Employers list don't match. Expected " + Arrays.toString(employers.toArray()) +
+        assertTrue("Employers list don't match. Expected " + Arrays.toString(applicantEmployers.toArray()) +
                         " but found " + Arrays.toString(ipsSearchPage.getEmploymentList().toArray()),
-                ipsSearchPage.getEmploymentList().containsAll(employers));
+                ipsSearchPage.getEmploymentList().containsAll(applicantEmployers));
     }
 
     @Test
     public void thatCatAPassingIndividualReturnsSuccess() throws IOException {
-        hmrcStub.stubCatAPassUser();
-        employers.add("Acme Inc");
-        submitValidApplicant(employers);
+        applicantEmployers.add("Acme Inc");
```

Comment:
Is this list cleared in between each test?

Reply:
no is not 😮

Reply:
I wonder how it works then :)